components/00-design-tokens/colors.config.yml

```diff
@@ -1,60 +1,57 @@
-context: 
+notes: "some notes"
```

Comment:
You don't need this line

Reply:
forgot i put that in there for testing 🤣